xbmc/addons/Skin.cpp

```diff
@@ -359,7 +359,8 @@ int CSkinInfo::GetFirstWindow() const
 bool CSkinInfo::IsInUse() const
 {
   // Could extend this to prompt for reverting to the standard skin perhaps
-  return CSettings::GetInstance().GetString(CSettings::SETTING_LOOKANDFEEL_SKIN) == ID();
+  bool active = GetRunningInstance() != nullptr;
```

Comment:
GetRunningInstance is a binary addon hack. For all other addons it's always null